src/main/java/github/grace000/erversay/Request/Request.java

```diff
@@ -0,0 +1,17 @@
+package github.grace000.erversay.Request;
```

Comment:
why the nesting if it's the same word?

Reply:
I'd been doing some file restructuring. I'll rename that folder.